bin/reth/src/node/mod.rs

```diff
@@ -143,19 +146,29 @@ impl Command {
         let network = self.start_network(network_config, &ctx.task_executor, ()).await?;
         info!(target: "reth::cli", peer_id = %network.peer_id(), local_addr = %network.local_addr(), "Connected to P2P network");
 
-        // TODO: Use the resolved secret to spawn the Engine API server
-        // Look at `reth_rpc::AuthLayer` for integration hints
-        let _secret = self.rpc.jwt_secret();
+        let test_transaction_pool = reth_transaction_pool::test_utils::testing_pool();
```

Comment:
@mattsse tagging that we'll want to init the real txpool here; to do after we're done w rpc